### PR TITLE
failure updating RN getting master/branch version error

### PR DESCRIPTION
**Pull request by gal-berger**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Description
1. fix getting pack_metadata.json on windows OS failure.
2. change error message from:
`Unable to locate the metadata on the master branch`
`The master branch is currently ahead of your pack's version. please pull from master and re-run the command`
to:
`"Unable to locate the metadata on the master branch.\n The reason is:<reason>"`
## Related Issues
fixes: https://github.com/demisto/etc/issues/28144

## Must have
- [ ] Test
- [ ] Documentation

### CODE REVIEW

File: demisto_sdk/commands/update_release_notes/update_rn.py [hunk 1]
```diff
@@ -41,6 +42,7 @@ def __init__(self, pack_path: str, update_type: Union[str, None], modified_files
 
         self.added_files = added_files
         self.pre_release = pre_release
+        self.pre_release = pre_release
```
Comment (amshamah419) [inline demisto_sdk/commands/update_release_notes/update_rn.py:45]: Probably don't need this twice

Comment (gal-berger) [reply demisto_sdk/commands/update_release_notes/update_rn.py]: fixed

**Review by amshamah419 — APPROVED** [review]
Just remember to make an addition in the changelog

File: demisto_sdk/commands/update_release_notes/update_rn.py [hunk 2]
```diff
@@ -134,13 +135,14 @@ def is_bump_required(self):
                 return False
             new_metadata = self.get_pack_metadata()
             new_version = new_metadata.get('currentVersion', '99.99.99')
-            master_metadata = run_command(f"git show origin/master:{self.metadata_path}")
+            master_metadata = run_command(
+                str(PurePosixPath(PureWindowsPath(f"git show origin/master:{self.metadata_path}"))))
```
Comment (yuvalbenshalom) [inline demisto_sdk/commands/update_release_notes/update_rn.py:139]: @gal-berger I think it is better to apply the path translation only on the self.metadata_path

**Review by yuvalbenshalom — CHANGES_REQUESTED** [review]
looks good,
see my comment, and plase add a change-log.

File: demisto_sdk/commands/update_release_notes/update_rn.py (hunk 2 above)
Comment (gal-berger) [reply demisto_sdk/commands/update_release_notes/update_rn.py]: pushed and tested using tox in both Linux and Windows.

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,7 @@
     * Added new argument *--id-set-path* for id_set.json path.
     * When changes to *APIModule* is detected and an id_set.json is available - the command will update the dependent pack as well.
 * Fixed an issue where the find_type function didn't recognize file types correctly.
+* Fix update-release-notes command on Windows OS.
```
Comment (yuvalbenshalom) [inline CHANGELOG.md:9]:
```suggestion
* Fix update-release-notes command on Windows.
```

**Review by yuvalbenshalom — CHANGES_REQUESTED** [review]

**Review by yuvalbenshalom — APPROVED** [review]

**Review by roysagi — APPROVED** [review]
seems good, please update the changelog with my suggestion.

**Pull request merged**